src/Sylius/Bundle/ApiBundle/CommandHandler/SendShopUserVerificationEmailHandler.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ApiBundle\CommandHandler;
+
+use Sylius\Bundle\ApiBundle\Command\SendShopUserVerificationEmail;
+use Sylius\Bundle\CoreBundle\Mailer\Emails;
+use Sylius\Component\Channel\Repository\ChannelRepositoryInterface;
+use Sylius\Component\Core\Model\ShopUserInterface;
+use Sylius\Component\Mailer\Sender\SenderInterface;
+use Sylius\Component\User\Repository\UserRepositoryInterface;
+use Symfony\Component\Messenger\Handler\MessageHandlerInterface;
+
+/** @experimental  */
+final class SendShopUserVerificationEmailHandler implements MessageHandlerInterface
```

Comment:
I would be for
```suggestion
final class SendCustomerVerificationEmailHandler implements MessageHandlerInterface
```
or
```suggestion
final class SendCustomerAccountVerificationEmailHandler implements MessageHandlerInterface
```
or even
```suggestion
final class SendAccountVerificationEmailHandler implements MessageHandlerInterface
```